docs/pages/by-operation.md

```
@@ -0,0 +1,111 @@
+import ScopeConfigurationTabs from '../.vitepress/theme/components/ScopeConfigurationTabs.vue'
+</script>
+
+# Pages by Operation
+
+You can use the `OAOperation` component to render a specific operation.
+
+## Creating `operations` pages
+
+To generate pages for each operation, create a directory named `operations` in the `docs` directory. Inside the `operations` directory, create a file named `[operationId].md` and a file named `[operationId].paths.js`.
+
+```
```

Markdownlint (0.37.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)